src/main.rs

```diff
@@ -113,8 +113,14 @@ fn main() {
     };
 
     // Initialize logger with level from config
-    simple_logger::init_with_level(carrot_config.logging().level().to_owned())
-        .expect("Failed to initialize logger");
+    let mut logger = simple_logger::SimpleLogger::new()
+        .with_utc_timestamps()
+        .with_level(carrot_config.logging().level().to_owned().to_level_filter());
+    // Add any module-specific levels
+    for item in carrot_config.logging().modules() {
+        logger = logger.with_module_level(item.0, item.1.to_owned().to_level_filter());
```

Comment:
Is item in this case a hash entry with key = item.0 and value = item.1?

Reply:
Yup, iterating over a hashmap like that gives you tuples with each key-value pair.  I could also write it like
```
for (key, value) in carrot_config.logging().modules() {
```
or
```
for (module, logging_level) in carrot_config.logging().modules() {
```
if you think that would be clearer.

Reply:
No, it's fine, I just wanted to make sure I understood what it was.